packages/vue/src/sdk.ts

```diff
@@ -61,6 +62,22 @@ Update your \`Sentry.init\` call with an appropriate config option:
   }
 }
 
+/**
+ * Initialize Vue-specific error monitoring for a given Vue app.
+ */
+export function initVueApp(app: Vue, client?: BrowserClient): void {
```

Comment:
I'm just wondering (we can add this later if we actually need it) but wdyt about optionally passing in the Vue-specific options that are used by `vueInit`? I'm thinking about a case where users would initialize the Browser SDK (or that would be done for them e.g. in the Astro SDK) and they'd additionally pass in the Vue options here.

Reply:
Hmm maybe actually instead of passing the client optionally, we should pass the options directly? 🤔 as we don't really _need_ the client in there, we just pick the options off...?